source/_authors/crell.twig

```diff
@@ -3,7 +3,7 @@ title: Larry Garfield
 identifier: crell
 twitter: Crell
 avatar: /img/blog/avatars/larry-garfield.jpg
-bio: Larry Garfield is an aspiring blacksmith who moonlights as Director of Developer Experience for Platform.sh.  When not trying to hand-forge his own medieval armory from scratch he tries to teach developers and development managers the skills of yesteryear that the industry has forgotten. #PSR8 compatible.
+bio: Larry Garfield is an aspiring blacksmith who moonlights as Director of Developer Experience for Platform.sh.  Since 2017 he has been a member of the FIG Core Committee. PSR-8 compatible.
```

Comment:
```suggestion
bio: Larry Garfield is an aspiring blacksmith who moonlights as Director of Developer Experience for Platform.sh.  Since 2017 he has been a member of the PHP-FIG Core Committee. PSR-8 compatible.
```